src/ui/Roster.ts

```diff
@@ -79,6 +84,78 @@ export default class Roster {
          spanElement.text(msg);
       }
    }
+   
+   private initCollapisible()
+   {
+      let coll = $(".jsxc-collapsible");
+      let i;
+
+      for (i = 0; i < coll.length; i++)
```

Comment:
`let` is block scope, so we can use it directly inside the loop declaration.

Reply:
fixed... now using for(let i;...